cmd/root.go

```diff
@@ -471,3 +480,49 @@ func defaultTemplatedHelp(cmd *cobra.Command, args []string) {
 		fmt.Fprintf(cmd.ErrOrStderr(), "unable to display help text: %v", err)
 	}
 }
+
+// Create and return a path to a temp dir prefixed 'func'.  Does not change
```

Comment:
@lkingland consider using `t.TempDir()` for temporary directories creation.

Reply:
Yes, thanks!  I changed to using this in most places